GliaWidgets/Sources/Component/Header/Header.Props.swift

```diff
@@ -0,0 +1,13 @@
+import UIKit
+
+extension Header {
+    struct Props {
```

Comment:
I would suggest to use constants for Props.

Reply:
until rework [EngagementViewController](https://github.com/salemove/ios-sdk-widgets/pull/485/files#diff-ae805a97a762b66a6e90c10388387a488324c2f29180305c3bd12b5e8f765fa0) it is not possible to make it `let`.